ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -127,20 +122,13 @@ const CancelSpeedupPopover = () => {
               </Box>
             }
           />
-        </Typography>
-        <div className="cancel-speedup-popover__separator" />
-        <Box
-          display={DISPLAY.FLEX}
-          alignItems={AlignItems.center}
-          flexDirection={FLEX_DIRECTION.COLUMN}
-          marginTop={4}
-        >
-          <Box className="cancel-speedup-popover__edit-gas-button">
-            {!appIsLoading && <EditGasFeeButton />}
-          </Box>
-          <Box className="cancel-speedup-popover__gas-details">
-            <GasDetailsItem />
-          </Box>
+        </Text>
+
+        <Box className="cancel-speedup-popover__edit-gas-button">
+          {!appIsLoading && <EditGasFeeButton />}
+        </Box>
+        <Box className="cancel-speedup-popover__gas-details">
+          <GasDetailsItem />
         </Box>
         <Button type="primary" onClick={submitTransactionChange}>
```

Comment:
The component-library Button is primary style by default
```suggestion
        <Button onClick={submitTransactionChange}>
```